src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -230,8 +246,9 @@ function PaymentMethodList(props) {
                 data={filteredPaymentMethods}
                 renderItem={renderItem}
                 keyExtractor={(item) => item.key}
-                ListEmptyComponent={renderListEmptyComponent(translate)}
+                ListEmptyComponent={props.shouldShowEmptyListMessage ? renderListEmptyComponent(translate) : null}
```

Comment:
At the top the props are being destructed, I would remove the `props` and add the `shouldShowEmptyListMessage`